thread_pool.c

```diff
@@ -60,7 +60,8 @@ static int worker_id(hts_tpool *p) {
 int DBG_OUT(FILE *fp, char *fmt, ...) {
     va_list args;
     va_start(args, fmt);
-    return vfprintf(fp, fmt, args);
+    int ret = vfprintf(fp, fmt, args);
```

Comment:
The `#else` dummy `DBG_OUT` is `void`, so really just discard this return value and change the function declaration to `void DBG_OUT(etc)`.